test/index.js

```diff
@@ -0,0 +1,18 @@
+var request = require('request'),
+    msg = process.env.TRAVIS_COMMIT_MSG;
```

Comment:
Ideally, yes. But it's not a quick one-liner. You'd have to spawn a child process to run the git command and that's more work than I wanted to do for this quick PR.